testing/internal/e2e/boundary/host.go

```diff
@@ -359,3 +359,53 @@ func WaitForNumberOfHostsInHostSetCli(t testing.TB, ctx context.Context, hostSet
 	)
 	require.NoError(t, err)
 }
+
+// CreateGCPHostCatalogCli uses the cli to create a new GCP dynamic host catalog.
+// Returns the id of the new host catalog.
+func CreateGCPHostCatalogCli(
```

Comment:
I *think* the convention is Gcp?
```suggestion
func CreateGcpHostCatalogCli(
```